libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.5.0.0
 
+* Add `getOriginalBytes`
```

Comment:
```suggestion
* Add `decodeAnnotated`
* Add `getOriginalBytes`
```
If I'm not mistaken, you need to mention `decodeAnnotated` here as well as that is also re-exported through `Cardano.Ledger.Binary.Decoding`, which is exposed.